### PR TITLE
update dockerfile and use prod mod by default

### PR DESCRIPTION
**Pull request by CedricCabessa**
Dockerfile now use wsgi in production mode
Developer who want to use the django webserver in dev mod simply need
to update settings.py to switch in DEBUG

### CODE REVIEW

**Comment by mmaret-geny** [comment]
LGTM

**Comment by jcleneveu-geny** [comment]
I'm not good at docker related thing but else it just LGTM

File: deploy/gm_pr.conf [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        <Directory /var/www/gm_pr/gm_pr>
+                <Files wsgi.py>
+                        Require all granted
+                </Files>
+        </Directory>
+</VirtualHost>
+
+WSGIScriptAlias / /var/www/gm_pr/gm_pr/wsgi.py
+WSGIPythonPath /var/www/gm_pr
+Alias /static/ /var/www/gm_pr/gm_pr/static/
```
Comment (fbarthelery) [inline deploy/gm_pr.conf]: missing new line

File: deploy/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FROM debian:jessie
+RUN apt-get update
+RUN apt-get install -y python3-django python3-django-celery celeryd python3-sqlalchemy supervisor apache2 libapache2-mod-wsgi-py3
+ADD gm_pr.conf /etc/apache2/sites-available/gm_pr.conf
+RUN a2ensite gm_pr
+EXPOSE 80
+WORKDIR /var/www/gm_pr
+CMD chown -R www-data:www-data . && supervisord -c deploy/supervisord.conf
```
Comment (fbarthelery) [inline deploy/Dockerfile]: you should put the chown comman in a RUN and keep CMD simple

**Comment by AFouques** [comment]
:hand:

File: deploy/Dockerfile (hunk 2 above)
Comment (CedricCabessa) [inline deploy/Dockerfile]: No I can't, /var/www/gm_pr is a shared volume

File: deploy/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,8 @@
+FROM debian:jessie
+RUN apt-get update
+RUN apt-get install -y python3-django python3-django-celery celeryd python3-sqlalchemy supervisor apache2 libapache2-mod-wsgi-py3
```
Comment (flo-genymobile) [inline deploy/Dockerfile]: i like having all package on a different line, i find it easier to read. 

```
RUN apt-get update && apt-get install -y \
  supervisor \
  python3-django \
  git 
```

File: deploy/gm_pr.conf (hunk 1 above)
Comment (CedricCabessa) [inline deploy/gm_pr.conf]: done

**Comment by flo-genymobile** [comment]
@CedricCabessa an answer to my comment would have been nice :)

**Comment by CedricCabessa** [comment]
@flo-genymobile sorry dude, I did'nt see it when I merged
yet another github bug

**Pull request merged**